src/main/java/edu/utdallas/davisbase/host/Host.java

```diff
@@ -39,9 +42,68 @@ public Host(HostConfiguration configuration, BufferedReader reader, PrintWriter
     this.writer = writer;
   }
 
-  public String readStatement() throws IOException {
-    // TODO Implement Host.readStatement()
-    throw new NotImplementedException();
+  static String prompt = "davisql> ";
+	static String copyright = "Team Blue";
+  static String version = "v1.0";
+  static boolean exit_prompt = false;
+
+	static Scanner scanner = new Scanner(System.in).useDelimiter(";");
+
+  public String readStatement(String input) throws IOException {
+
+    PromptScreen();
+
+		String userInput = "";
+
+		System.out.print(prompt);
+    userInput = scanner.next().replace("\n", " ").replace("\r", "").trim().toLowerCase();
+    return userInput;
+  }
+
+
+  public static void PromptScreen() {
+		System.out.println(line_chars("-",80));
+    System.out.println("Welcome to DavisBase");
+		System.out.println("DavisBase Version " + version);
+		System.out.println(copyright);
+		System.out.println("\nType \"help;\" to display supported commands.");
+		System.out.println(line_chars("-",80));
+	}
+
+
+	public static String line_chars(String s,int num) {
+		String a = "";
+		for(int i=0;i<num;i++) {
+			a += s;
+		}
+		return a;
+  }
+
+  public String display_fix(int len, String s) {
+		return String.format("%-"+(len+3)+"s", s);
+	}
+
+
+	public static void help() {
+		System.out.println(line_chars("*",80));
+		System.out.println("SUPPORTED COMMANDS");
+		System.out.println("All commands below are case insensitive");
+		System.out.println();
+		System.out.println("\tSHOW TABLES;                                                 Display all the tables in the database.");
+		System.out.println("\tCREATE TABLE table_name (<column_name datatype>);            Create a new table in the database.");
+		System.out.println("\tINSERT INTO table_name VALUES (value1,value2,..);            Insert a new record into the table.");
+		System.out.println("\tDELETE FROM TABLE table_name WHERE row_id = key_value;       Delete a record from the table whose rowid is <key_value>.");
```

Comment:
Remove this line for now. (but save it for later) Part 1 of the project isn't supposed to support the `DELETE` command.